NEWS.md

```diff
@@ -157,6 +157,17 @@ To be released as 2.3.0
 
 ### Layers: geoms, stats, and position adjustments
 
+* `stat_contour()`, `stat_density2d()`, `stat_bin2d()`,  `stat_binhex()` now
+  include  a normalized version the `level`, `density`, and `count` parameters, 
+  named `nlevel`, `ndensity`, and `ncount`. This mirrors the use of `ndensity`
+  in `stat_density()` and `ncount`/`ndensity` in `stat_bin()`. These stats
+  should be useful for faceted 2D density plots and bin plots in cases where
+  the distribution of density within a facet is more relevant than the absolute
+  density or count across facets. (@bjreisman, #2680)
+
+* `stat_density()` now includes the calculated statistic `nlevel`, an alias 
+  for `scaled`, to better match the syntax of `stat_bin` (@bjreisman, #2680)
+
```

Comment:
Unfortunately this whole part seems to have ended in the wrong section of the file. It should be at the very top, under ggplot2 3.0.0.9000. You may have to rebase. Also, the whole section should only be 1-2 sentences, such as "`stat_contour()`, `stat_density2d()`, `stat_bin2d()`,  `stat_binhex()` all now also calculate normalized densities, levels, and counts (@bjreisman, #2680)".

Reply:
Apologies, one more comment: The issue this is closing is #2679, not #2680.  #2680 is the pull request. And one of the git commit messages should contain the phrase "Closes #2679" or "Fixes #2679" so that github automatically closes the issue once the pull request is integrated. You can add this to the next commit you make.